### DISCUSSION TITLE
Activate by a certain shortcut

### DISCUSSION THREAD

**Discussion by pacifier007** [espanso/espanso#1265] [Feature requests and Ideas]
Is there a way for the Espanso to act, passively only activated when a shortcut is used?

I don't want it to be always active and only activate when needed. 

Previously I was using a Chrome browser exetension that worked like this:
I typed a word and press Ctrl + Space and it would fill in the text for that word. The automatic mode can be bit incorrect sometimes and I don't want it using resources always reading and parsing text. It would be better to activate it when needed.

**Comment by federico-terzi** [comment]
@pacifier007 Thank you for the feedback! We'll investigate, in the meantime, you can disable espanso and re-enable when needed: https://espanso.org/docs/configuration/options/#customizing-the-toggle-key
